### PR TITLE
pycodestyle 2.2.0 (new formula)

### PR DESCRIPTION
**Pull request by straxhaber**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by ilovezfs — CHANGES_REQUESTED** [review]
Test is failing https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2047/version=sierra/testReport/junit/brew-test-bot/sierra/test_pycodestyle/

File: Formula/pycodestyle.rb
Comment (bfontaine) [inline Formula/pycodestyle.rb]: The `test` block is here to check everything’s got installed and is running correctly; not to run the project’s test suite.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by JCount** [comment]
@straxhaber Ping on this

**Comment by straxhaber** [comment]
I have made changes to the formula that fix the issue. How should I go about submitting it? Do I just push a second commit on top of this one, or is there something else I should do?

**Comment by JCount** [comment]
@straxhaber For now, just pushing the second commit to your branch is probably best.

**Comment by straxhaber** [comment]
How is it now? Did I do the correct thing?

**Comment by straxhaber** [comment]
oops, one sec -- I need to fix it

File: Formula/pycodestyle.rb
Comment (bfontaine) [inline Formula/pycodestyle.rb]: Please write a test that does something more substantial than e.g. `--help` or `--version`. You could write a dummy Python file and assert its output on it. Something like so:
```ruby
(testpath/"dummy.py").write <<-EOS.undent
  print("hello I'm a Python file")
  # ...
EOS

assert_equal "...expected output...",
    shell_output("#{bin}/pycodestyle.py ... dummy.py")
```

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by straxhaber** [comment]
How's that?

**Review by bfontaine — APPROVED** [review]

**Comment by straxhaber** [comment]
@ilovezfs just waiting on your approval. Is there anything else I should change/fix?

File: Formula/pycodestyle.rb
Comment (ilovezfs) [inline Formula/pycodestyle.rb]:
```
bin.install "pycodestyle.py" => "pycodestyle"
```

**Review by ilovezfs — CHANGES_REQUESTED** [review]
I'd also like to see the test block test for a style violation. Thanks!

**Comment by straxhaber** [comment]
@ilovezfs How's that?

**Comment by straxhaber** [comment]
Also, @ilovezfs @bfontaine would you rather I keep submitting additional commits or squash and force push? If the former, should I make the commit message the standardized format as I have been or make more meaningful commit messages?

**Comment by bfontaine** [comment]
@straxhaber It’d be great if you could squash into one commit with the standardized format and force-push, thanks 😃

**Comment by straxhaber** [comment]
@bfontaine @ilovezfs squashed!

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @straxhaber! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by straxhaber** [comment]
@ilovezfs! Thanks!

**Pull request closed without merging**
